src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = {};
+  // const properties = [];
+  // const styles = [];
+  const stylesArr = sourceString.split(';');
+
+  for (const property of stylesArr) {
+    const splittesProperty = property.split(':');
+
+    if (splittesProperty[0].trim().length
+    && splittesProperty[1].trim().length > 0) {
```

Comment:
it's better to use array destructuring here, for instance:
`const [key, value] = splittesProperty`;